charts/pega/values.yaml

```diff
@@ -110,11 +110,29 @@ global:
         # Inactivity time after which requestor is passivated
         passivationTimeSec: 900
 
+
       service:
         # For help configuring the service block, see the Helm chart documentation
         # https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/README.md#service
         port: 80
         targetPort: 8080
+        tlsport: 443
+        tlstargetPort: 8443
+
+      # to enable TLS between ingress/loadbalancer and the backend, set the following:
+      tls:
```

Comment:
We have another 'tls' in ingress, can we rename this to something else to avoid confusion?